### PR TITLE
docs: Update devel readme

### PR DESCRIPTION
**Pull request by aesteve-rh**
Update devel README file to reflect
latest updates with sample-images
and automotive-image-builder commands
and project structures.

### CODE REVIEW

**Review by sandrobonazzola — APPROVED** [review]
LGTM

File: docs/devel/README.md [hunk 1]
```diff
@@ -100,18 +100,18 @@ The following commands will execute:
   - ssh enabled
 
 ```bash
-dnf install podman -y
+dnf install podman -y && dnf clean all
 git clone https://gitlab.com/CentOS/automotive/sample-images.git
-cd sample-images/osbuild-manifests/
-dnf clean all
+git submodule update --init
```
Comment (Yarboa) [inline docs/devel/README.md:105]: I would redirect here to the relevant link in automotive sig, in case it will change in the future

Comment (Yarboa) [reply docs/devel/README.md:105]: Please ignore, already exist above

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**